test/core/ref_func.wast

```diff
@@ -0,0 +1,55 @@
+(module
+  (func (export "f") (param $x i32) (result i32) (local.get $x))
+)
+(register "M")
+
+(module
+  (func $f (import "M" "f") (param i32) (result i32))
+  (func $g (param $x i32) (result i32) (i32.add (local.get $x) (i32.const 1)))
+
+  (global anyref (ref.func $f))
+  (global anyref (ref.func $g))
+  (global funcref (ref.func $f))
+  (global funcref (ref.func $g))
+  (global $x (mut funcref) (ref.func $f))
```

Comment:
rename so it isn't confused with various `param $x`

Reply:
Done.